app-backend/db/src/test/scala/com/azavea/rf/database/CampaignDaoSpec.scala

```diff
@@ -250,4 +251,69 @@ class CampaignDaoSpec
       )
     }
   }
+
+  test("list campaigns by continent") {
+    check {
+      forAll(
+        (
+            userCreate: User.Create,
+            campaignCreates1: List[Campaign.Create],
+            campaignCreates2: List[Campaign.Create]
+        ) => {
+          val pageSize = 30
+          val pageRequest = PageRequest(0, pageSize, Map.empty)
+          val continents = List(
+            Continent.Asia,
+            Continent.Africa,
+            Continent.Antarctica,
+            Continent.Australia,
+            Continent.Europe,
+            Continent.NorthAmerica,
+            Continent.SouthAmerica
+          )
+          val continent = continents((new Random).nextInt(continents.size))
```

Comment:
You won't need this if you provide an `Arbitrary[Continent]` using the `Gen[Continent]` you already wrote